src/core/server/services/i18n/index.ts

```diff
@@ -109,6 +110,10 @@ export function translate(
 ): string {
   const message = bundle.getMessage(id);
   if (!message) {
+    if (config.get("env") === "testing") {
```

Comment:
https://github.com/coralproject/talk/blob/next/src/core/server/config.ts#L48
I think it should check for `"test"`